backend/prisma/schema.prisma

```diff
@@ -7,31 +7,110 @@ generator prisma_client {
   provider = "prisma-client-js"
 }
 
-enum Status {
-  ADMIN MODERATOR FREE_USER PRO_USER
+model User {
+  Id                String        @id @default(cuid())
+  Name              String
+  Email             String        @unique
+  Password          String
+  ResetToken        String?
+  ResetTokenExpiry  Float?
+  userRole          UserRole      @default(FREE_USER)
+  CreatedAt         DateTime      @default(now())
+  UpdatedAt         DateTime      @updatedAt
+  Handle            String        @unique
+  Bio               String
+  SocialMedia       SocialMedia[]
+  interests         Topic[]
+  location          Location?
+  languagesNative   Language[]    @relation("LanguageToNative")
+  languagesLearning Language[]    @relation("LanguageToLearner")
 }
 
-model User {
-  Id               String   @id @default(cuid())
-  Name             String
-  Email            String   @unique
-  Password         String
-  ResetToken       String?
-  ResetTokenExpiry Float?
-  status           Status[]
-  posts            Post[]
-  CreatedAt        DateTime @default(now())
-  UpdatedAt        DateTime @updatedAt
+model SocialMedia {
+  Id       String              @id @default(cuid())
+  Platform SocialMediaPlatform
 }
 
 model Post {
-  Id         String   @id @default(cuid())
+  Id         String     @id @default(cuid())
   Title      String
   Body       String
-  Published  Boolean
+  status     PostStatus @default(DRAFT)
   Image      String?
```

Comment:
Yep yep, I remember this was a good suggestion you'd had previously back when we were doing the previous iteration. I didn't add it yet because I need to think a little about the best way to construct it, but it's on the list for this PR for sure! Will update you when that's ready to take another look at 🏎💨✨